alacritty_terminal/src/term/mod.rs

```diff
@@ -1422,10 +1422,15 @@ impl<T: EventListener> Handler for Term<T> {
                 if self.mode.contains(TermMode::ALT_SCREEN) {
                     self.grid.reset_region(..);
                 } else {
+                    let old_offset = self.grid.display_offset();
+
                     self.grid.clear_viewport();
 
+                    // How many lines has been scrolled by clearing viewport.
```

Comment:
I don't like comments which are phrased as questions rather than statements. I think it's more appropriate to have something like `Compute number of lines scrolled by clearing the viewport`.